src/core/exception.d

```diff
@@ -47,29 +47,81 @@ else:
  */
 class RangeError : Error
 {
-    @safe pure nothrow this( string file = __FILE__, size_t line = __LINE__, Throwable next = null )
+    // lower/upper bound passed, if slice (array[lower .. upper]), if not a slice (array[upper])
```

Comment:
Is there a reason these are not Ddoc comments?

Reply:
Because I suck at DDOC and this is still a WIP ;)

Reply:
It doesn't have the `wip` label 😉.

Reply:
OK good to know.